### PR TITLE
Adding the #interact( EntityPlayer, Vector ) method for entities

### PR DESCRIPTION
**Pull request by KCodeYT**

### CODE REVIEW

File: gomint-server/src/main/java/io/gomint/server/network/handler/PacketInventoryTransactionHandler.java [hunk 1]
```diff
@@ -147,6 +147,8 @@ private void handleUseItemOnEntity( io.gomint.entity.Entity target, PlayerConnec
                     break;
                 }
 
+                entity.interact( connection.getEntity() );
```
Comment (geNAZt) [inline gomint-server/src/main/java/io/gomint/server/network/handler/PacketInventoryTransactionHandler.java]: The vector is missing

**Review by geNAZt — CHANGES_REQUESTED** [review]

File: gomint-server/src/main/java/io/gomint/server/network/handler/PacketInventoryTransactionHandler.java (hunk 1 above)
Comment (KCodeYT) [reply gomint-server/src/main/java/io/gomint/server/network/handler/PacketInventoryTransactionHandler.java]: fuck xd

File: gomint-server/src/main/java/io/gomint/server/entity/Entity.java [hunk 2]
```diff
@@ -1105,6 +1105,10 @@ public boolean damage( EntityDamageEvent damageEvent ) {
         return !damageEvent.isCancelled();
     }
 
+    public void interact( io.gomint.entity.EntityPlayer player, Vector clickVector ) {
```
Comment (lukeeey) [inline gomint-server/src/main/java/io/gomint/server/entity/Entity.java:1108]: This can be declared abstract.

Comment (geNAZt) [reply gomint-server/src/main/java/io/gomint/server/entity/Entity.java:1108]: yeah it should be abstract

Comment (KCodeYT) [reply gomint-server/src/main/java/io/gomint/server/entity/Entity.java:1108]: no it cant if i make it abstract every entity have to implement this method but not every entitay need that

**Review by geNAZt — APPROVED** [review]

**Pull request merged**